openedx/license-manager
# feat: adds a LicenseTransferJob model and admin

Also installs django-autocomplete-light for help to support advanced admin autocomplete field filtering, and therefore runs make upgrade to pull in some other new package versions.

## Description
Adds a model for a "job" to transfer licenses from one plan to another within the same customer agreement.

Link to the associated ticket: https://openedx.atlassian.net/browse/ENT-7704

## Testing considerations
1. Create a customer agreement and two plans related to it.
2. For one of the plans, modify some of the license states to assigned or active.  Give them distinct, fake email addresses.
3. Create a `LicenseTransferJob`: http://localhost:18170/admin/subscriptions/licensetransferjob/ Optionally use the `is_dry_run` option to see which licenses _would_ be moved, without actually moving them.
4. After creating the plan, use the admin list view to select your record, then choose the "process" action and click "Go"
![image](https://github.com/openedx/license-manager/assets/2307986/6d142f2f-ea96-48ee-b528-116314196d54)

If not a dry-run, notice that the licenses have changed subscription plans.
